react/features/browser-capabilities/capabilities.json

```diff
@@ -0,0 +1,71 @@
+{
+    "Safari": [
```

Comment:
I don't see neither Elector nor mobile mentioned in this file. Is it handled as one of the browsers or it's not going to be used ?

Reply:
What is the purpose of this API ?

Reply:
The purpose is to provide a way for external apps to be able to check if a browser is supported by Jitsi Meet and which features are going to work. For example if somebody is using the iframe API and want to implement some logic for unsupported browsers.